schema.sql

```diff
@@ -11,3 +11,23 @@ CREATE TABLE animals (
 /* Day two the used queries */
 
 ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
+/* Day three the used QUERIES */
+
+-- add owners table
+
+CREATE TABLE owners ( id SERIAL PRIMARY KEY, full_name VARCHAR(255), age INTEGER )
+
+--add species table
+
+CREATE TABLE species ( id SERIAL PRIMARY KEY, name VARCHAR(255) )
+
+--modify animals table
+
+ALTER TABLE animals
+  DROP COLUMN species,
+  ADD COLUMN species_id INTEGER REFERENCES species(id)
+  ADD COLUMN owner_id INTEGER REFERENCES owners(id)
```

Comment:
- [ ] Kindly add a `comma(,)` after the `ADD COLUMN species_id INTEGER REFERENCES species(id)` line to show that the next statement is another substatement in the general statement. The `comma(,)` acts as a separator of substatements in a large SQL statement. Kindly use it to avoid syntax errors. 👍 😎 Please, do not forget to use the semi-colon(;) at the end of the general statement. 🌵

Reply:
Thanks!